libs/dh/market-participant/feature-edit-actor/src/lib/market-participant-edit-actor/contact-data/dh-market-participant-actor-contact-data.component.ts

```diff
@@ -83,22 +74,21 @@ export class DhMarketParticipantOrganizationContactDataComponent
     })
   );
 
-  contactRows: EditableContactRow[] = [];
-  deletedContacts: ContactDto[] = [];
+  contactRows: EditableActorContactRow[] = [];
+  deletedContacts: ActorContactDto[] = [];
 
   ngOnChanges() {
```

Comment:
Suggestion: Add argument for `ngOnChanges` like example below, and adjust the `if statement` only to care about if `contacts` has changed.   

[ngOnChanges(changes: SimpleChanges): void](https://angular.io/api/core/OnChanges#ngOnChanges)